apps/prairielearn/src/lib/client/page-context.test.ts

```diff
@@ -57,7 +57,7 @@ describe('getPageContext', () => {
 });
 
 describe('getCourseInstanceContext', () => {
-  const mockStudentData: StudentCourseInstanceContext = {
+  const mockStudentData: Omit<StudentCourseInstanceContext, '__brand'> = {
```

Comment:
Of course, you can no longer construct this object yourself (which is a good thing!)